packages/radio-group/stories/radio-group.stories.mdx

```diff
@@ -21,7 +21,7 @@ You should use `<lion-radio>`s inside this element.
 </Story>
 
 ```html
-<lion-radio-group name="dinos" label="What are your favourite dinosaurs?">
```

Comment:
This is just the code snippet that the user can copy right? So here, I wouldn't change it. I would only change it in the <Story> sections.

Reply:
For me, we should keep the same. As there is an understanding of the reader that the snippet of code will result in exactly what he sees. But i can change it if you think it's better for readability.

Reply:
I think for most developers they will just (blindly) copy paste the code to get a working radio group and then change the name to their own context so I guess it doesn't matter either way. I see your point that it's more 1 to 1 to keep it like this 👍 .